### PR TITLE
Typos fixed and updated to latest state

### PR DESCRIPTION
**Pull request by claudiow**
The typo is fixed.

The appveyor-mvn-conversion.yml is nor redundant. It will become the release script once I have everything ready.

This will be a build that is triggered manually by you whenever a new release is ready.

Right now it differs from appveyor.yml on that isntead of running `mvn clean install` it runs `mvn clean site deploy`

When we are ready the regular build will do this, build and deploy the SNAPSHOT with the latest changes with every commit, which keeps that repository updated for the brave that want to use the latest development version.

BTW we have our SNAPSHOT deployed to Sonatype's repository check it [https://oss.sonatype.org/content/groups/public/org/dblock](https://oss.sonatype.org/content/groups/public/org/dblock)

### CODE REVIEW

**Comment by claudiow** [comment]
I'm taking care of the conflicts, I let you know when I'm done

**Comment by claudiow** [comment]
Done

**Comment by claudiow** [comment]
Don't forget to keep your secret during the merge.

**Comment by dblock** [comment]
Lets fix this git history - you see the many many merge commits here. This is because you work off master and it's out of sync with the parent.

I think you should start with this.

Make sure you have an upstream remote, so:

```
git add remote upstream git@github.com:dblock/log4jna.git
```

Then put your changes on a branch.

```
git checkout master
git checkout -b fix-yml # creates a fix-yml branch with all the changes currently on master
git checkout master
git reset HEAD~100 --hard # rewinds master back 100 commits, loses all changes
git pull upstream master # syncs up master
```

Now, adjust the fix-yml branch.

```
git checkout fix-yml
git rebase master # rebase against master
```

Push `fix-yml` to Github.

```
git push origin fix-yml
```

Make a new pull request from that branch, close this one.

File: appveyor.yml [hunk 1]
```diff
@@ -21,8 +21,12 @@ environment:
   SEC_KEY_FILE_GPG: $(APPVEYOR_BUILD_FOLDER)\build\Maven\secring.gpg.asc
   SECURE_FILE_DIR: C:\secure-file
 
+  # For use in Claudio's development environment testing
```
Comment (dblock) [inline appveyor.yml]: Can this just be an environment variable set on the AppVeyor website?

**Comment by claudiow** [comment]
Well no mater what I do I can't get the build in your AppVeyor environment working. I added your secret value commented and my build works. see: [https://ci.appveyor.com/project/claudiow/log4jna](https://ci.appveyor.com/project/claudiow/log4jna)

You will have to merge and un-comment your %secret% variable.
.
Unfortunately because I'm testing the appveyor configuration this is unavoidable, This is going away once I have Maven deployment working and we work with your build only.

**Comment by claudiow** [comment]
I wonder if I will be able to fix this if you create a team and add me to it in your AppVeyor environment.

**Comment by dblock** [comment]
Are you sure @claudiow? What if I set my secret in my log4jna appveyor configuration to my value and you set it to yours? For example:

```
secret:
   secure: %SECRET%
```

(maybe pick a better name)

**Comment by dblock** [comment]
Can you also explain what appveyor-mvn-conversion.yml vs. appveyor.yml?

**Comment by claudiow** [comment]
Actually I do all the work on a branch `maven-conversion`. When I have something ready for merge I merge into my fork `master`

This is because I don't freeze work in the branch as its is a painstaking trial and error process to have maven doing what I want in an environment that is designed for MSFT development.

The branch is already ahead of this PR as I continue testing maven deployment to central.

**Comment by claudiow** [comment]
I will test if setting the secret  variable in the configuration works.

**Comment by claudiow** [comment]
The `appveyor-mvn-conversion` differs from `appmveyor.yml` in that it will be the configuration for Maven Releases to Central.

We will have a manually triggered build that you will be able to run whenever a new release is ready.

It will look something like this:
```yaml
build: off # Disable automatic builds
  - mvn release:prepare

on_success:
  - mvn release:perform

on_failure:
  - mvn releas:roolback
``` 

This will take care of:

1. Updating version numbers in the pom, 
2. Create a tag in git
3. Check out the tag from git
4. Build jars and site on the tag
5. Deploy the build to Maven Central
6. Deploy the site to its  web site
7. Check in the POM with an incremented SNAPSHOT version number into `master`

In the meantime, allows me to test and fail and modify the configuration without disturbing te working appveyor.yml that is working for `master` 

I merge from `appveyor-mvn-conversion` into `appmveyor.yml` when I have a working milestone to deploy.

**Comment by claudiow** [comment]
BTW All this annoyances around the CI exists only because I'm working and testing AppVeyor configuration. 

I created my own project at AppVeyor under my own account as I don't have other means of testing what I'm doing, I have to run the builds see them fail or succeed and even echo stuff to the log in orther to discover how to fit the square in the circle.

All this should go away once we have a working release and you configure it in the project CI.

**Pull request closed without merging**